xcode/Subconscious/Shared/Components/Common/SubtextView.swift

```diff
@@ -79,7 +79,6 @@ struct SubtextView: View {
                 }
             }
         }
-        .transition(.slide)
```

Comment:
Unrelated change for issue discovered while troubleshooting the SubtextView. We should not be decorating SubtextView with transition(.slide). This is a general-purpose view, and slide is a very opinionated transition (slides left to right).

Reply:
I'm not even sure why this is here in the first place, any transition should be on the container...